src/sentry/eventstore/models.py

```diff
@@ -519,7 +519,7 @@ def get_raw_data(self, for_stream: bool = False) -> Mapping[str, Any]:
 
     @property
     def size(self) -> int:
-        return len(json.dumps_experimental("eventstore.enable-orjson", dict(self.data)))
+        return len(orjson.dumps(dict(self.data)).decode())
```

Comment:
Can we get rid of `dict()` call? It makes an unnecessary copy

```suggestion
        return len(orjson.dumps(self.data).decode())
```